homeassistant/components/wemo.py

```diff
@@ -75,21 +76,41 @@ def coerce_host_port(value):
 
 def setup(hass, config):
     """Set up for WeMo devices."""
+    _LOGGER.debug("Beginning setup of WeMo component...")
```

Comment:
We can remove this. The core already logs how fast each component was set up and when.